artman/tasks/package_metadata_tasks.py

```diff
@@ -94,12 +95,42 @@ def _write_yaml(self, config_dict, dest):
         with open(dest, 'w') as f:
             yaml.safe_dump(config_dict, f, default_flow_style=False)
 
-class ProtoPackageMetadataGenTask(task_base.TaskBase):
-    default_provides = 'proto_pkg_dir'
+class JavaGrpcPackageMetadataConfigGenTask(PackageMetadataConfigGenTask):
```

Comment:
I was trying to group the `config gen task` and `metadata gen task` :)
See L24 and L128